client/my-sites/plugins/plan-setup/index.jsx

```diff
@@ -42,13 +42,15 @@ module.exports = React.createClass( {
 	},
 
 	componentDidMount() {
+		if ( ! this.props.selectedSite || ! this.props.selectedSite.jetpack || ! this.props.selectedSite.canManage() ) {
+			return;
```

Comment:
This needs to be the most non-error-feeling page ever. It's not so much an error as just 'the next step' I think.